rules/teams-group-chat/rule.md

```diff
@@ -0,0 +1,29 @@
+---
+type: rule
+title: Do you create a group chat with your project team?
+uri: teams-group-chat
+authors:
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+created: 2023-01-24T05:56:17.498Z
+guid: 1ad391a4-19c8-4dc9-82fb-b83a6f08074f
+---
+
+Teams is a powerful way to chat, collaborate, and video call with your project team. As soon as you start a new project, create a group chat with all the team members (potentially including the client if appropriate).
+ 
+Although you will also have a dedicated Team (as per SSW.Rules | Do you know how to create a Team?), which is great for storing documents and files, as well as persisting notifications, you’re better to communicate in a chat rather than a channel since:
```

Comment:
```suggestion
Although you will also have a dedicated Team (as per [https://www.ssw.com.au/rules/create-a-team/](SSW.Rules | Do you know how to create a Team?)), which is great for storing documents and files, as well as persisting notifications, you’re better to communicate in a chat rather than a channel since:
```